.travis.yml

```diff
@@ -29,29 +25,32 @@ addons:
       - libssl-dev
       - libstdc++6
       - binutils-gold
-      # Provides a backtrace if the unittests crash
-      - gdb
-      # needed to build gdb
-      - texinfo
+      - cmake
+      - lcov
+      - llvm-5.0
+      - clang-5.0
 
 matrix:
   include:
     # Default BUILD is "scons".
 
+    - compiler: gcc
+      env: GCC_VER=5 BUILD=cmake TARGET=debug PATH=$PWD/cmake/bin:$PATH APP_ARGS="--unittest-jobs=2"
+
     # - compiler: gcc
     #   env: GCC_VER=5 TARGET=debug.nounity
 
-    - compiler: gcc
-      env: GCC_VER=5 BUILD=cmake TARGET=coverage PATH=$PWD/cmake/bin:$PATH
+    # - compiler: gcc
+    #   env: GCC_VER=5 BUILD=cmake TARGET=coverage PATH=$PWD/cmake/bin:$PATH
 
-    - compiler: clang
-      env: GCC_VER=5 BUILD=cmake TARGET=debug CLANG_VER=3.8 PATH=$PWD/llvm-$LLVM_VERSION/bin:$PWD/cmake/bin:$PATH
-      cache:
-        directories:
-          - $GDB_ROOT
+    # - compiler: clang
+    #   env: GCC_VER=5 BUILD=cmake TARGET=debug CLANG_VER=3.8 PATH=$PWD/llvm-$LLVM_VERSION/bin:$PWD/cmake/bin:$PATH
```

Comment:
Maybe add a note here about `CLANG_VER` since we're bringing in 3.8 now (I know this is commented out, but I'd still like a note)

Reply:
I don't think `CLANG_VER` needs to be there at all anymore...

Reply:
I verified in another branch that the clang build works fine without all these extra variables? https://travis-ci.org/ximinez/rippled/builds/346626338?utm_source=github_status&utm_medium=notification

And I notice it seems to complete pretty quickly. Do you think it's worth including both builds?